### PR TITLE
docs: fix typos

### PR DESCRIPTION
**Pull request by deining**
This PR fixes various typos I spotted in the project.

### CODE REVIEW

File: cli/tsc/00_typescript.js [hunk 1]
```diff
@@ -56362,7 +56362,7 @@ function createTypeChecker(host) {
         if (
           // Check for unusable parameters symbols
           symAtLocation === unknownSymbol || // If the symbol is not found, but was not found in the original scope either we probably have an error, don't reuse the node
-          symAtLocation === void 0 && sym !== void 0 || // If the symbol is found both in declaration scope and in current scope then it shoudl point to the same reference
+          symAtLocation === void 0 && sym !== void 0 || // If the symbol is found both in declaration scope and in current scope then it should point to the same reference
```
Comment (dsherret) [inline cli/tsc/00_typescript.js:56365]: Can you revert these ones? It's in generated code.

File: cli/tsc/dts/lib.dom.d.ts [hunk 2]
```diff
@@ -10918,7 +10918,7 @@ interface HTMLIFrameElement extends HTMLElement {
      */
     src: string;
     /**
-     * Sets or retrives the content of the page that is to contain.
+     * Sets or retrieves the content of the page that is to contain.
```
Comment (dsherret) [inline cli/tsc/dts/lib.dom.d.ts:10921]: Same here.

File: cli/tsc/dts/typescript.d.ts [hunk 3]
```diff
@@ -1808,7 +1808,7 @@ declare namespace ts {
                 includeLinePosition?: boolean;
             }
             /**
-             * Response object for synchronous sematic diagnostics request.
+             * Response object for synchronous semantic diagnostics request.
```
Comment (dsherret) [inline cli/tsc/dts/typescript.d.ts:1811]: And here.

File: cli/tsc/00_typescript.js (hunk 1 above)
Comment (deining) [reply cli/tsc/00_typescript.js]: Reverted my change as requested.

File: cli/tsc/dts/typescript.d.ts (hunk 3 above)
Comment (deining) [reply cli/tsc/dts/typescript.d.ts]: Reverted my change as requested.

File: cli/tsc/dts/lib.dom.d.ts (hunk 2 above)
Comment (deining) [reply cli/tsc/dts/lib.dom.d.ts]: Reverted my change as requested.

**Review by dsherret — APPROVED** [review]
Thanks!

**Pull request merged**